lib/data_processing.py

```diff
@@ -35,6 +35,10 @@ def load_data(input_file='input_delivery.csv', input_path="output"):
     # import data and fill nulls with 0
     df = pd.read_csv(os.path.join("..",input_path, input_file)).fillna(0)
 
+    # fill unknown ethnicity from GP records with ethnicity from SUS (secondary care)
+    df.loc[df["ethnicity"]==0, "ethnicity"] = df["ethnicity_6_sus"]
+    df.loc[df["ethnicity_16"]==0, "ethnicity_16"] = df["ethnicity_16_sus"]
+    
```

Comment:
It would be good to allow option of handling this processing in the python study definition

Reply:
Agree - it is currently cumbersome in study definition as the logic has to be written for each possible value - a new function would make this easier but there's a bit of discussion around this. Issue [here](https://github.com/opensafely-core/cohort-extractor/issues/487).